scripts/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+var slideshow = [];
+
+function Slide (slideObject) {
+
+  this.title = slideObject.title;
```

Comment:
This is great data to add to your individual projects. Now you just need to create the data and add it in along with removing this from the slideshow and having it appear in its own portfolio area.